resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -188,6 +188,12 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Allocation batch delay must be a positive time value.")
       .createWithDefaultString("1s")
 
+  val KUBERNETES_MAX_EXECUTOR_FAILURES =
+    ConfigBuilder("spark.kubernetes.max.executor.failures")
+      .doc("The maximum number of executor failures before failing the application.")
+      .intConf
```

Comment:
I think we could use a checkValue here for a positive number

Reply:
yea